src/backend/optimizer/path/joinpath.c

```diff
@@ -2318,6 +2317,7 @@ hash_inner_and_outer(PlannerInfo *root,
 			 */
 			if (innerrel->partial_pathlist != NIL &&
 				save_jointype != JOIN_UNIQUE_INNER &&
+				save_jointype != JOIN_LASJ_NOTIN &&
```

Comment:
Just move one line? Great!

Reply:
Yeah. Enable parallel-oblivious and disable parallel-aware.